### ISSUE TITLE
Something wrong with my experiment configuration, I got the result of dropping 10 points compared with paper in semi-supervisied OD.

### ISSUE THREAD

**Issue by Wastoon** [soo89/CSD-RFCN#1]
What a wonderful work it is! Thanks for your paper and open source code! Unfortunately, when I try the code, I cannot get the results wrote in the paper, could you share the specific configuration when training the semi-supervised model, especially the backbone of feature extraction network, looking forward to your reply. Thank you very much!

**Comment by soo89** [comment]
Have you checked the score with supervised learning?
(I have checked it several times, and there is a slight variation but it is not much with your score. )

I wonder if you use a pre-trained model.

**Comment by Wastoon** [comment]
> Have you checked the score with supervised learning?
> (I have checked it several times, and there is a slight variation but it is not much with your score. )
> 
> I wonder if you use a pre-trained model.

Yes, I also checked the score with supervised learning, the score also drops nearly 10 points, so I doubt that something must be wrong with the type of feature extraction backbone I use, and I used the pre-trained model. Could you provide the info of feature extraction backbone type and supervised loss type? Thanks for your reply!

**Comment by soo89** [comment]
I have no idea.

I didn't change the code after the experiments and upload it.

I used 4 GPUs and 4 minibatch with the ResNet101 model.

Have you evaluated the last model that is trained? 

Please check in the test_net.py code.

**Comment by Wastoon** [comment]
> I have no idea.
> 
> I didn't change the code after the experiments and upload it.
> 
> I used 4 GPUs and 4 minibatch with the ResNet101 model.
> 
> Have you evaluated the last model that is trained?
> 
> Please check in the test_net.py code.
That makes sense, my results are from ResNet50, I will retrain it on ResNet101. Thank you very much, wonderful work!
